frontend/projects/upgrade/src/app/features/dashboard/segments/modals/upsert-private-segment-list-modal/upsert-private-segment-list-modal.component.scss

```diff
@@ -0,0 +1,14 @@
+:host ::ng-deep .upsert-private-segment-list-container {
+  display: flex;
+  flex-direction: column;
+  align-items: space-between;
+  width: 100%;
+
+  .mat-mdc-form-field {
+    width: 100%;
+  }
+
+  .mat-mdc-form-field:not(:last-child) {
+    margin-bottom: 20px;
+  }
+}
```

Comment:
I think it's OK to remove the whole style from this file. The `upsert-private-segment-list-container` selector doesn't exist anyway.